static_src/actions/user_actions.js

```diff
@@ -119,6 +120,65 @@ const userActions = {
     });
   },
 
+  fetchUserInvite(email) {
+    AppDispatcher.handleViewAction({
+      type: userActionTypes.USER_INVITE_FETCH,
+      email
+    });
+
+    return uaaApi.inviteUaaUser(email)
+      .then(data => userActions.receiveUserInvite(data));
+  },
+
+  receiveUserInvite(inviteData) {
+    AppDispatcher.handleServerAction({
+      type: userActionTypes.USER_INVITE_RECEIVED
+    });
+
+    const userGuid = inviteData.new_invites[0].userId;
+
+    return cfApi.postCreateNewUserWithGuid(userGuid)
+      .then(user => userActions.receiveUserInCF(user, inviteData));
+  },
+
+  receiveUserInCF(user, inviteData) {
```

Comment:
Reconfirmed @jcscottiii. The create user happens in the function above. This is the function that takes the created user and then processes the return data (`invite_url`/`user_guid`) and takes the next steps.